src/com/facebook/buck/android/AndroidResource.java

```diff
@@ -157,6 +158,16 @@
 
   private final AtomicReference<String> rDotJavaPackage;
 
+  private final Function<HasAndroidResourceDeps, Path> resourceDirectoryFunction =
+      new Function<HasAndroidResourceDeps, Path>() {
+        @Override
+        public Path apply(HasAndroidResourceDeps input) {
+          return Optional.fromNullable(input.getRes())
+              .transform(getResolver().getPathFunction())
+              .get();
+        }
+      };
+
```

Comment:
Thanks for the advice.  I'll have a look at `AndroidLibraryGraphEnhancer.getBuildableForAndroidResources` and see if I can cook something up over the next few days!